pkg/cloud_provider/clientset/clientset.go

```diff
@@ -115,11 +115,13 @@ func (c *Clientset) ConfigurePodLister(nodeName string) {
 
 		nodeName := podObj.Spec.NodeName
 		volumes := podObj.Spec.Volumes
+		restartPolicy := podObj.Spec.RestartPolicy
```

Comment:
Do you also need to specify status fields?

Reply:
We are only filtering out fields inside Pod.Spec. Pod.Status is unaffected.